src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -629,12 +629,21 @@ private static String getMessageBundle(String className) {
         final String messageBundle;
         final String messages = "messages";
         final int endIndex = className.lastIndexOf('.');
+        final Map<String, String> messageBundleMappings = new HashMap<>();
+        messageBundleMappings.put("SeverityMatchFilterExamplesTest",
+                "com.puppycrawl.tools.checkstyle.checks.naming.messages");
+
         if (endIndex < 0) {
             messageBundle = messages;
         }
         else {
             final String packageName = className.substring(0, endIndex);
-            messageBundle = packageName + "." + messages;
+            if ("com.puppycrawl.tools.checkstyle.filters".equals(packageName)) {
+                messageBundle = messageBundleMappings.get(className.substring(endIndex + 1));
```

Comment:
Please change logic to use value from manual mapping if mapping is present.
If present in special map then use it , else use packageName